easybuild/framework/easyconfig/tools.py

```diff
@@ -623,6 +623,14 @@ def categorize_files_by_type(paths):
         # file must exist in order to check whether it's a patch file
         elif os.path.isfile(path) and is_patch_file(path):
             res['patch_files'].append(path)
+        elif path.endswith('.patch'):
```

Comment:
They will fail to parse and error out as before. However those are uncommon enough to just do that.

Of course we could check for `.eb` extensions (and maybe easystack? what is their ext?) and error out here for anything else. But that would be a breaking change in case anyone (for whatever reason) uses `.ec` for their easyconfigs

Reply:
Easystack files currrently need to be passed as an argument to `--easystack`, so that outside this scope (at least currently, that may change).

Maybe we can start using `.ebs` or `.es` as extension for easystack files.